alibi/confidence/model_linearity.py

```diff
@@ -218,7 +219,7 @@ def _sample_knn(x: np.ndarray, X_train: np.ndarray, nb_samples: int = 10) -> np.
     return np.asarray(X_sampled)  # shape=(nb_instances, nb_samples, nb_features)
 
 
-def _sample_grid(x: np.ndarray, feature_range: np.ndarray = None, epsilon: float = 0.04,
+def _sample_grid(x: np.ndarray, feature_range: np.ndarray, epsilon: float = 0.04,
```

Comment:
There is no `None` handling in the function so `feature_range` must be required.